### ISSUE TITLE
Instructions to setup oobabooga backend

### ISSUE THREAD

**Issue by albertopirillo** [rizerphe/obsidian-companion#22]
It seems that, even if not mentioned everywhere in the readme, oobabooga support was added in #13 by @kabachuha.
Apparently you guys managed to get it working (correct me if I am wrong), but unfortunately I cannot.

Could you provide some instructions to link obsidian-companion with text generation webui? 
I have a working setup of oobabooga with a model loaded which I can use to chat. 
I have started the OpenAI-compatible oobabooga API with the flags `--api` `--listen`. 

However, I do not understand how to configure the Companion from Obsidian settings. 
I have already tried with multiple IPs into the API URL field, but none seems to work.

**Comment by Alexandrsv** [comment]
<details><summary>oobabooga  is boken</summary>
![image](https://github.com/rizerphe/obsidian-companion/assets/15097064/e79d12b4-6779-4a80-a9ac-74416fcc400d)

</details>

**Comment by albertopirillo** [comment]
Yes, and that's because HTTP requests are being sent to incorrect URLs.
Take a look at [this code](https://github.com/rizerphe/obsidian-companion/blob/main/src/complete/completers/oobabooga/oobabooga.tsx), which is supposed to retrieve completions from the oobabooga API.
The URLs are way different from the ones in the [oobabooga's API guide](https://github.com/oobabooga/text-generation-webui/wiki/12-%E2%80%90-OpenAI-API#examples). 
Maybe they were made for an older version and were never updated.
